src/components/NavBar.js

```diff
@@ -34,12 +39,26 @@ const NavBar = props => {
               >
                 Projects
               </Button>
-              <Button onClick={() => props.history.push("/pending-projects")}>
-                Pending Projects
-              </Button>
-              <Button onClick={() => props.history.push("/project-form")}>
-                Submit Project
-              </Button>
+
+              {user ? (
+                user.role==="admin" ? (
+                    <Button
+                      className="pendingProjectsButton"
+                      onClick={() => props.history.push("/pending-projects")}
+                    >
+                      Pending Projects
+                    </Button>
+                  ) : (
+                    <Button
+                      className="projectFormButton"
+                      onClick={() => props.history.push("/project-form")}
+                    >
+                      Submit Project
+                    </Button>
+                  
+                )
+              ) : null }
```

Comment:
Is this last `: null` line doing anything?

Reply:
yeah, it's not rendering either when you're not logged in. so they cant see pending and they cant submit apps